test/sslapitest.c

```diff
@@ -9410,6 +9410,66 @@ static int test_sni_tls13(void)
     SSL_CTX_free(cctx);
     return testresult;
 }
+
+/*
+ * Test that the lifetime hint of a TLSv1.3 ticket is no more than 1 week
+ * 0 = TLSv1.2
+ * 1 = TLSv1.3
+ */
+static int test_ticket_lifetime(int idx)
+{
+    SSL_CTX *cctx = NULL, *sctx = NULL;
+    SSL *clientssl = NULL, *serverssl = NULL;
+    int testresult = 0;
+    unsigned char data[16] = { 0 };
+    int version = TLS1_3_VERSION;
+
+#define TWO_WEEK_SEC (14 * 24 * 60 * 60)
+#define ONE_WEEK_SEC (7 * 24 * 60 * 60)
```

Comment:
I'd suggest having this as:

```c
#define ONE_WEEK_SEC (7 * 24 * 60 * 60)
define TWO_WEEK_SEC (2 * ONE_WEEK_SEC)
```

Not critical of course.

Reply:
Done